src/features/emails/types.ts

```diff
@@ -165,7 +165,7 @@ export type BlockAttributes = {
   }>;
 };
 
-export type EmailFrame = {
+export type EmailTheme = {
   block_attributes?: BlockAttributes;
   css?: string;
   frame_mjml: MJMLJsonObject | null;
```

Comment:
I think `fram_mjml` should also be renamed to `theme_mjml!`

Reply:
I disagree about this. IMO, a theme is three things: The frame, the styling and the default block attributes. The `frame_mjml` property is the MJML for the frame part of the theme. Before the CSS and the block attributes were introduced, this thing that we used to call the frame was only a frame, but then it became more, which is why I think it's good to have renamed it to theme. But I don't think we should rename that part that is actually a frame.

Are you convinced? 😊

Reply:
You have convinced med! :D